_posts/0000-01-13-martinquenneville.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "This is Martin Quenneville's slide deck!"
```

Comment:
@martinquenneville  Perhaps you would prefer not using your name?

Reply:
But i have no imagination

Reply:
OK I will approve anyway :smiley: